src/course-unit/course-xblock/CourseXBlock.test.jsx

```diff
@@ -290,4 +290,22 @@ describe('<CourseXBlock />', () => {
       expect(mockedUsedNavigate).toHaveBeenCalledWith(`/course/${courseId}/editor/problem/${id}`);
     });
   });
+
+  it('displays a visibility message if item has accessible restrictions', async () => {
+    const { getByText } = renderComponent(
+      {
+        userPartitionInfo: {
+          ...camelCaseObject(userPartitionInfo),
```

Comment:
[important] It looks like we can convert each keys into camelCase in `courseVerticalChildrenMock` file instead using camelCaseObject every time.